src/lib/useDragAndZoom.tsx

```diff
@@ -92,6 +94,7 @@ export function useDragAndZoom(
     onHover?: (x: number, event: PointerEvent) => void,
     onHoverEnd?: () => void,
     onTouchUp?: (x: number, event: PointerEvent) => void,
+    onChangeBounds?: (start: number, end: number) => void,
```

Comment:
Do you really need a new callback here? Doesn't onManipulationEnd match your needs?